hosting/learnings-from-2025.md

```diff
@@ -0,0 +1,80 @@
+# Learnings from the 2025 Hackathon
+
+All in all, this was surprisingly successful. There were some technical issues though.
+
+## Overall organization
+
+* Andrew, Pier-Luigi, Sara and Yuting did a great job at holding things together
+* Bjorn did a great job at shaking up structures and pushing things forward
+* Tobi, Lukas, Mark M, and others made technical magic possibble.
+
+## Schedule
+
+## Inter-node communication
+
+### Mattermost
+
+* Works
+* Invite more people earlier in the process
+* China uses WeChat for most communication (in Chinese, but with auto-translation)
+
+### Zoom calls during the preparation phase
+
+* Double Telcos are tiring but necessary.
+* Did anybody read the nodes / watch the recordings?
+
+### Zoom calls during the hackathon
+
+* Did not really line up with the schedules (at least at some nodes). Would need earlier planning to be incorporated into the local schedules.
```

Comment:
I absolutely agree. If one really wants these update calls they have to be included in all plans and their goal needs to be clearly defined (see comment above)